tests/constraint_tests.py

```diff
@@ -178,6 +178,30 @@ def test_linear_transformer_invest(self):
 
         self.compare_lp_files("linear_transformer_invest.lp")
 
+    def test_nonconvex_invest_transformer(self):
+        """Non convex invest flow with offset, without minimum."""
+        bfuel = solph.buses.Bus(label="fuelBus")
+        bel = solph.buses.Bus(label="electricityBus")
+
+        solph.components.Transformer(
+            label="transformer_nonconvex_invest",
+            inputs={bfuel: solph.flows.Flow()},
+            outputs={
+                bel: solph.flows.NonConvexInvestFlow(
+                    nominal_value=None,
+                    variable_costs=25,
+                    min=0.2,
+                    max=0.8,
```

Comment:
Technically speaking, it is possible to have integer numbers >= 1 also for min/max. But i.e. 0.25 should also work. (It's just a hint. I thought it's nice to know. We also have numbers with ugly binary representation is the other tests. I just try to avoid this whenever I am aware that this might be a thing.)

Reply:
Ok then we will use 0.25 and 0.5 :)